pytest_mh/utils/fs.py

```diff
@@ -611,3 +611,25 @@ def chmod(self, mode: str, path: str, args: list[str] = []) -> SSHProcessResult:
         self.logger.info(f'Changing mode bits for "{path}" on {self.host.hostname}')
 
         return self.host.ssh.exec(["chmod", *args, mode, path])
+
+    def chown(self, path: str, user: str | None = None, group: str | None = None, args: str = "") -> SSHProcessResult:
+        """
+        Change file owner and group.
+
+        :param path: path to file
+        :type path: str
+        :param user: new file owner, if None then user remains same, defaults to None
+        :type user: str | None, optional
+        :param group: new file group, if None then group remains same, defaults to None
+        :type group: str | None, optional
+        :param args: additional arguments, defaults to ""
+        :type args: str, optional
+        :return: Result of process
+        :rtype: SSHProcessResult
+        """
+        self.backup(path)
+        self.logger.info(f'Changing owner for "{path}" on {self.host.hostname}')
+
+        mode = f"{user}" if user else ""
+        mode = mode + f":{group}" if group else mode
+        return self.host.ssh.run(f"chown {mode} {path} {args}")
```

Comment:
Please, use args as list and exec here as we do on other places (chmod as example).

Reply:
Okay, fixed